common/src/main/java/com/wynntils/models/character/CharacterWardrobeModel.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.character;
+
+import com.wynntils.core.components.Handlers;
+import com.wynntils.core.components.Model;
+import com.wynntils.handlers.actionbar.event.ActionBarUpdatedEvent;
+import com.wynntils.mc.event.ContainerClickEvent;
+import com.wynntils.models.character.actionbar.matchers.CharacterWardrobeSegmentMacher;
+import com.wynntils.models.character.actionbar.segments.CharacterWardrobeSegment;
+import com.wynntils.models.characterstats.actionbar.segments.HealthBarSegment;
+import java.util.List;
+import net.neoforged.bus.api.SubscribeEvent;
+
+public class CharacterWardrobeModel extends Model {
+    private boolean inWardrobe = false;
+    private boolean afterAnimation = false;
+
+    public CharacterWardrobeModel() {
+        super(List.of());
+        Handlers.ActionBar.registerSegment(new CharacterWardrobeSegmentMacher());
+    }
+
+    @SubscribeEvent
+    public void onActionBarUpdate(ActionBarUpdatedEvent event) {
+        event.runIfPresent(CharacterWardrobeSegment.class, this::enableInWardrobe);
+        event.runIfPresent(HealthBarSegment.class, this::disableInWardrobe);
+    }
+
+    @SubscribeEvent
+    public void ContainerClickEvent(ContainerClickEvent event) {
```

Comment:
```suggestion
    public void onContainerClick(ContainerClickEvent event) {
```

Reply:
Do you even need this though? Is the segment not good enough for detection?

Reply:
I wanted to make it smooth. The hud is hidden as soon as the container is closed, then it goes into an animation and when it finishes, it sends the action bar. I wanted to match that behavior as much as I could. Altho for some reason and 1 time only, it didn't register the click, I never could manage to repeat that so just in case I made the action bar matcher to also set it.